packages/x-date-pickers-pro/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import type {} from '@mui/material/themeCssVarsAugmentation';
```

Comment:
```suggestion
```

This **should not** be enabled by the library, it should be done by the consumer instead.

Can you try importing it into the MUI X global project instead?

Reply:
what do you mean by
> importing it into the MUI X global project

Reply:
Without it, TypeScript complains about `theme.vars` as it's not available on the standard `Theme` interface.
Can we solve it somehow without importing `themeCssVarsAugmentation`?

Reply:
Even if we fix it on our side, make sure to test a bundled version of our package with `skipLIbCheck: false` because I suspect it will fail without this import.

Doesn't it make sense for `@mui/x-date-pickers` to import the module augmentation that it uses internally ?

Reply:
> Doesn't it make sense for @mui/x-date-pickers to import the module augmentation that it uses internally

I am open to any option but the developers have to opt-in to get the type. Meaning when they install `@mui/x-date-pickers` and `@mui/material`, they should not see `theme.vars`.

If they want to have `theme.vars`, they have to opt-in (either import from `@mui/material` or `@mui/x-date-pickers`).

Reply:
I struggle to understand how we could use `theme.vars` in our files, and hide the `theme.vars` type from users